### ISSUE TITLE
IF END-IF without no instruction is OK for zOpenEditor

### ISSUE THREAD

**Issue by ROBIN-Olivier** [IBM/zopeneditor-about#341]
- Z Open Editor version: 3.1.1
- Editor Platform
  - [x] Visual Studio Code 1.72.2
- Operating System: Windows 10 Entreprise
- Java Version: 11

When I code a condition with IF ... END-IF without statement, compilation is KO, but parsing of zOpenEditor is OK:
![image](https://user-images.githubusercontent.com/103997342/236411183-a1bbb65c-fabc-42d4-a1ec-281a09429720.png)

**Comment by phaumer** [comment]
Changed it to an enhancement as our editor will not catch everything that the compiler rejects.

**Comment by FALLAI-Denis** [comment]
Hi,

As I already said, [The language server is not a language compiler](https://github.com/IBM/zopeneditor-about/discussions/184) 😊

**Comment by phaumer** [comment]
I agree with that. Our goal is to, of course, not show any false errors for programs that compile, but we cannot show all errors the compiler would find. However, we can only add value if we can make that set of errors we do show in the editor as large as we can. Hence, this would be a valuable enhancement.

**Comment by ROBIN-Olivier** [comment]
The developers I train are delighted that this parsing shows them syntax errors, so they avoid a lot of compilation errors when writing their instructions.

I understand that the parser cannot catch **all** compilation errors as it depends on the compiler.
But are there compilers that accept an empty IF END-IF (#341) or a COPY in column 7 (#340)?
